runtime/lua/vim/lsp/buf.lua

```diff
@@ -827,11 +827,19 @@ local function on_code_action_results(results, opts)
     return
   end
 
-  ---@param item {action: lsp.Command|lsp.CodeAction}
+  ---@param item {action: lsp.Command|lsp.CodeAction, ctx: lsp.HandlerContext}
   local function format_item(item)
```

Comment:
Thinking ahead I wonder if it would be more flexible to provide a `format_item` callback setting to let users customize each action as their heart desire instead of having a bunch of boolean settings.

Reply:
Definitely, or even a more generalized solution for overriding handlers. But this PR doesn't add a new boolean flag.

Reply:
I would be happy to implement this in the current or a future PR (whatever is most appropriate).

Reply:
it's a bigger topic, see https://github.com/neovim/neovim/issues/25259#issuecomment-1727336789